Site/ASAB Maestro/Versions/v25.08.dev1.yaml

```diff
@@ -0,0 +1,47 @@
+define:
+  type: rc/version
+  product: ASAB Maestro
+  version: v25.08.dev1
```

Comment:
Likely unrelated by try to use anchor for version so that you don't need to repeat it at `library asab-maestro-library: v25.08.dev1`

Reply:
:+1: ok, i dont know, how and where to categorize this note. but yes, let's improve the version files :slightly_smiling_face:

Reply:
I created the version file just to test the bootstrap. (So I can use the favor of CI/CD)